app/Resources/views/pagelayout_footer.html.twig

```diff
@@ -1,6 +1,42 @@
-<footer>
-    <p>Copyright &copy; 1999-{{ "now"|date("Y", "Europe/Oslo") }} eZ Systems AS.</p>
-    <p>
-        {{ ez_render_field( content, "footer_ez_links" ) }}
-    </p>
+<footer id="footer">
+    <div class="footer-content">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-3">
+                    <img src="{{ asset('assets/images/ez-logo.svg') }}" alt="eZ Systems" class="logo" />
+                </div>
+
+                <div class="col-md-3">
+                    {{ ez_render_field( content, "footer_ez_links" ) }}
+                </div>
+
+                <div class="col-md-3">
+                    {{ ez_render_field( content, "footer_doc_links" ) }}
+                </div>
+
+                <div class="col-md-3 footer-social">
+                    <h4>{{ 'Follow us on social media'|trans }}</h4>
+
+                    <p>
+                        <a href="//www.youtube.com/user/ezpublish/" target="_blank"><i class="fa fa-youtube fa"> </i></a>
+                        <a href="//google.com/+ezsystems" target="_blank"><i class="fa fa-google-plus fa"> </i></a>
+                        <a href="//www.linkedin.com/company/16738" target="_blank"><i class="fa fa-linkedin fa"> </i></a>
+                        <a href="//www.facebook.com/eZPublishEnterprise" target="_blank"><i class="fa fa-facebook fa"> </i></a>
+                        <a href="//twitter.com/#!/ezsystems" target="_blank"><i class="fa fa-twitter fa"> </i></a>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="copyright-content">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12 text-center">
+                    <p>Copyright &copy; 1999-{{ "now"|date("Y", "Europe/Oslo") }} eZ Systems AS. All rights reserved.</p>
```

Comment:
single quotation marks in Twig helpers
you can remove second argument for the `date` filter (`"Europe/Oslo"`)